org-ql.el

```diff
@@ -1501,7 +1506,7 @@ Multiple predicates are combined with BOOLEAN."
                                      positive-term))
                            (positive-term (or (and predicate-with-args `(pred args -- (cons (intern pred) args)))
                                               (and predicate-without-args `(pred -- (list (intern pred))))
-                                              (and plain-string `(s -- (list 'regexp s)))))
+                                              (and plain-string `(s -- (list org-ql-default-predicate s)))))
```

Comment:
That plain strings in queries are currently searched with `regexp` applies not just to the non-Lisp query syntax but to Lisp queries as well.  For example:

```el
(org-ql--pre-process-query "string")  ;;=> (regexp "string")
```

I will need to look at the code more carefully, because I think this `peg` expression may need to be changed anyway.  It's probably not necessary (or, with this proposed change, a good idea) to add the predicate symbol here.  However, it's possible that doing so was the only way I could get the PEG expression to work properly--much trial-and-error was involved.

Reply:
Sorry I do not think I under stand this piece :-)

Reply:
Basically, this is not the correct (or only) place where the plain-string predicate is applied.  See, e.g. `org-ql--pre-process-query`.